src/h5web/toolbar/NxSpectrumToolbar.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { ReactElement } from 'react';
+import { MdFitnessCenter } from 'react-icons/md';
+import { useNxSpectrumConfig } from '../visualizations/nexus/spectrumConfig';
+import ToggleBtn from './controls/ToggleBtn';
+import LineToolbar from './LineToolbar';
+
+function NxSpectrumToolbar(): ReactElement {
+  const { showErrors, toggleErrors, areErrorsDisabled } = useNxSpectrumConfig();
+
+  return (
+    <LineToolbar>
+      <ToggleBtn
+        label="Error bars"
+        icon={(props) => <MdFitnessCenter transform="rotate(45)" {...props} />}
```

Comment:
:neutral_face:  ...

Taking proposals for the icon :rofl:

Reply:
What about:

![image](https://user-images.githubusercontent.com/2936402/99782850-fc2e4680-2b19-11eb-84a2-d684d7f14da4.png)

Reply:
I think the label of the toggler could also just be `Errors`